fiftyone/operators/executor.py

```diff
@@ -646,7 +646,21 @@ def current_sample(self):
     @property
     def user_id(self):
         """The ID of the user executing the operation, if known."""
-        return self.user.id if self.user else None
+        return self._user.id if self._user else None
+
+    @property
+    def user(self):
+        """An object of information about the user executing the operation, if
+        known.
+        """
+        return self._user
+
+    @property
+    def user_request_token(self):
+        """The request token authenticating the user executing the operation,
```

Comment:
It's a property so you can't edit it unless there's a corresponding setter.
Why the plugin dev would need to use the V51 request token I'm not sure

Reply:
Note that `ctx.user` is always `None` in OSS.

I added this property here because it already does exist in Teams and I want `ExecutionContext` to have the same contract in both OSS and Teams so that plugin developers can write plugins that are compatible with both OSS and Teams without hacky things like `if hasattr(ctx, attr)` clauses.